met-web/package.json

```diff
@@ -8,6 +8,7 @@
     "@emotion/styled": "^11.8.1",
     "@formio/react": "^5.2.1",
     "@mui/icons-material": "^5.8.3",
+    "@mui/lab": "^5.0.0-alpha.91",
```

Comment:
can you try a release version instead of alpha version?

Reply:
Tab context is a new material ui component and i dont think there is a release version for mui/lab